packages/ui-utils/src/global.css

```
@@ -378,30 +410,55 @@
 
   body {
     @apply bg-page-background text-foreground text-base;
-    font-feature-settings: 'rlig' 1, 'calt' 1;
+    font-family: var(--font-family-text);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)